gamestonk_terminal/stocks/technical_analysis/ta_controller.py

```diff
@@ -1110,6 +1112,64 @@ def call_bbands(self, other_args: List[str]):
         except Exception as e:
             print(e, "\n")
 
+    def call_donchian(self, other_args: List[str]):
+        """Process donchian command"""
+        parser = argparse.ArgumentParser(
+            add_help=False,
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+            prog="donchian",
+            description="""
+                Donchian Channels are three lines generated by moving average
+                calculations that comprise an indicator formed by upper and lower
+                bands around a midrange or median band. The upper band marks the
+                highest price of a security over N periods while the lower band
+                marks the lowest price of a security over N periods. The area
+                between the upper and lower bands represents the Donchian Channel.
+                """,
+        )
+        parser.add_argument(
+            "-lu",
+            "--length_upper",
+            action="store",
+            dest="n_length_upper",
+            type=check_positive,
+            default=20,
+            help="length",
+        )
+        parser.add_argument(
+            "-ll",
```

Comment:
Last comment:  For argparse convention: a single dash is followed by a single letter, so can you have the arguments be something like `-u` for upper and `-l` for lower